package.json

```diff
@@ -6,6 +6,11 @@
     "@testing-library/jest-dom": "^5.16.3",
     "@testing-library/react": "^12.1.4",
     "@testing-library/user-event": "^13.5.0",
+    "@types/jest": "^28.1.6",
+    "@types/node": "^18.0.6",
+    "@types/react": "^18.0.15",
+    "@types/react-dom": "^18.0.6",
+    "axios": "^0.27.2",
```

Comment:
@MatthewBerglund did you use `Axios` before? do you have any strong feelings (pro/cons) compared to the simple `fetch()`?

Reply:
@GiulianoMontisEvernest The `Expense` unit tests are run in the node testing environment because jest will not interact with the firestore emulator properly unless run this way, so this rules out `fetch()` as an option.

Reply:
ah ok! didn't know that thx! I thought that node also supports fetch() now: https://blog.logrocket.com/fetch-api-node-js/ but I'm not familiar with jest, so please disregard if not relevant